source/fundamentals/crud/query-document.txt

```diff
@@ -34,13 +34,16 @@ of query operators in MongoDB and show examples on how to use them:
 
 - :manual:`Evaluation Operators </reference/operator/query-evaluation/>`
 
-Use the following code snippet to create a collection of documents that
-describe an inventory of fruit to follow along with our query operator
-examples:
+Use the following code snippet to insert documents that
+describe an inventory of fruit to the ``myDB.fruits`` collection to
```

Comment:
S: Later examples just say "documents describing books" or "documents with restaurant data." It might be easier to remove the inventory concept here, unless it's helping to explain something

```suggestion
Use the following code snippet to insert documents that
describe fruits to the ``myDB.fruits`` collection to
```

Reply:
changed on 3 pages